src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/Serialization/MessagePack/Formatters/RazorConfigurationFormatter.cs

```diff
@@ -31,6 +31,7 @@ public override RazorConfiguration Deserialize(ref MessagePackReader reader, Ser
         var useConsolidatedMvcViews = reader.ReadBoolean();
         var useRoslynTokenizer = reader.ReadBoolean();
         var csharpLanguageVersion = (LanguageVersion)reader.ReadInt32();
+        var rootNamespace = reader.ReadString();
```

Comment:
Use the `CachedStringFormatter` as it did before.

```suggestion
        var rootNamespace = CachedStringFormatter.Instance.Deserialize(ref reader, options);
```

Reply:
Thank you, I should absolutely have done this.